### PR TITLE
How did it start - add the radio buttons forms 

### PR DESCRIPTION
**Pull request by khalidelaggan**
# Description

This is the new addition of the missing forms 
- How did it start form 
- How many times did it happen form

Fixes # (issue)

> Added the drafts for the radio buttons

- [ ] New feature

# Checklist:

Notes:;

My changes generate a new warnings relative to the array list and will be fixing this in my subsequent PR

TODO:
- Add french content 
- Add testing

### CODE REVIEW

File: f2/src/forms/BusinessInfoForm.js [hunk 1]
```diff
@@ -49,11 +49,11 @@ export const BusinessInfoForm = props => {
             )}
           </Field>
           <P>
-            <Trans id="businessInfoPage.nextPage" />
+            <Trans id="whatWasAffected.nextPage" />
           </P>
 
           <NextAndCancelButtons>
-            <Trans id="businessInfoPage.nextButton" />
+            <Trans id="whatWasAffected.nextButton" />
```
Comment (sastels) [inline f2/src/forms/BusinessInfoForm.js]: I don't think these two should change?

File: f2/src/forms/HowDidItStartForm.js [hunk 2]
```diff
@@ -127,10 +183,53 @@ export const HowDidItStartForm = props => {
...
+            </Control>
+            <P>
+              <Trans id="whatWasAffectedForm.nextPage" />
+            </P>
+
             <NextAndCancelButtons>
-              <Trans id="howDidItStartPage.nextButton">
-                Next: ??
-              </Trans>
+              <Trans id="whatWasAffectedForm.nextButton" />
```
Comment (sastels) [inline f2/src/forms/HowDidItStartForm.js]: Should this be `howDidItStartPage.nextButton`?

Comment (khalidelaggan) [reply f2/src/forms/HowDidItStartForm.js]: Changes are now embedded into the last commit + page links to the proper workflow

File: f2/src/locales/fr.json [hunk 3]
```diff
@@ -167,24 +201,14 @@
...
-  "evidencePage.intro": "Ce n'est pas grave si vous ne conservez aucune documentation de l'événement. Vous pouvez passer à la section suivante.",
-  "evidencePage.details": "Pensez à inclure des détails tels que :",
-  "evidencePage.detail1": "Captures d'écran de pages Web, conversations ou fils de discussion.",
-  "evidencePage.detail2": "Images de reçus, de transactions ou de bons de livraison.",
-  "evidencePage.detail3": "Enregistrement des demandes formulées par le suspect.",
-  "evidencePage.addFileButtom": "Ajouter un fichier",
-  "evidencePage.removeFileButton": "Supprimer le fichier",
-  "evidencePage.nextButton": "continuer",
-  "evidencePage.backButton": "Retour à qui pourrait être impliqué",
-  "evidencePage.nextPage": "Ensuite, il vous sera demandé votre emplacement.",
```
Comment (khalidelaggan) [inline f2/src/locales/fr.json]: Little concerned about this as I have to run the extract command, hoping no translation is lost for other dev :(

Comment (khalidelaggan) [reply f2/src/locales/fr.json]: @sastels  I have synced my local files with master and ran npm extract so this should not cause any conflicts when merged anymore

File: f2/src/forms/HowDidItStartForm.js [hunk 4]
```diff
@@ -91,7 +147,7 @@ export const HowDidItStartForm = props => {
 
                 {questionsList.map((question) => {
                   return (
-                    <Box question={question.channel}>
+                    <Box key={question.channel}>
```
Comment (khalidelaggan) [inline f2/src/forms/HowDidItStartForm.js]: Required- >Each child in an array should have a unique "key" prop fix

File: f2/src/HowDidItStartPage.js [hunk 5]
```diff
@@ -39,7 +39,7 @@ export const HowDidItStartPage = () => {
             <HowDidItStartForm
               onSubmit={data => {
                 dispatch({ type: 'saveFormData', data: { howdiditstart: data } })
-                history.push(doneForms ? '/confirmation' : '/WhatWasAffectedPage')
+                history.push(doneForms ? '/privacyconsent' : '/whatwasaffected')
```
Comment (sastels) [inline f2/src/HowDidItStartPage.js]: I think this should just be 
```
history.push(doneForms ? '/confirmation' : '/whatwasaffected')
```

File: f2/src/PrivacyConsentPage.js [hunk 6]
```diff
@@ -37,7 +37,7 @@ export const PrivacyConsentPage = () => {
                   type: 'saveFormData',
                   data: { consent: data },
                 })
-                history.push('/whatwasaffected') // TODO: change this to /howdiditstart when that page is done
+                history.push('/howdiditstart') // TODO: change this to /howdiditstart when that page is done
```
Comment (sastels) [inline f2/src/PrivacyConsentPage.js]: can delete the comment 😄

Comment (khalidelaggan) [reply f2/src/PrivacyConsentPage.js]: ooops done :)

File: f2/src/locales/fr.json (hunk 3 above)
Comment (sastels) [reply f2/src/locales/fr.json]: looks like you still have those lines in line 50 - 61 of your file, so they were duplicated.

**Pull request merged**